pkg/allocateip/allocateip.go

```diff
@@ -136,10 +173,10 @@ func assignHostTunnelAddr(ctx context.Context, c client.Interface, nodename stri
 	attrs := map[string]string{ipam.AttributeNode: nodename}
 	var handle string
 	if vxlan {
-		attrs[ipam.AttributeType] = "vxlanTunnelAddress"
+		attrs[ipam.AttributeType] = vxlanIPAMAttrString
```

Comment:
Huh, we actually already have constants defined for this here: https://github.com/projectcalico/libcalico-go/blob/master/lib/ipam/ipam.go#L46-L47

Reply:
Done.